md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/LineageAPI.java

```diff
@@ -117,30 +117,28 @@ RestWrapper getByBatch(@PathVariable("batchId") String batchId, Principal princi
                         "<TR><TD>Start Time:</TD></TR>");
 
                 if (startTime == null) {
-                    InstanceExecLabel.append("<TR><TD> NA </TD></TR>");
+                    instanceExecLabel.append("<TR><TD> NA </TD></TR>");
                 } else {
-                    InstanceExecLabel.append("<TR><TD>" + format.format(startTime) + "</TD></TR>");
+                    instanceExecLabel.append("<TR><TD>" + format.format(startTime) + "</TD></TR>");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "</TD></TR>" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "<TR><TD>" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)